okta-hosted-login/okta-aspnetcore-mvc-example/Program.cs

```diff
@@ -8,18 +8,19 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 
+#pragma warning disable SA1300 // Element should begin with upper-case letter
 namespace okta_aspnetcore_mvc_example
```

Comment:
Should we rename it to have the suggested capitalization?

Reply:
That's a good question! I kept using this "format" to be consistent with the existent samples we had. But I am happy to change it if you all agree 😄 . @nbarbettini any preference?